plugins/gazebo_server/GazeboServer.cc

```diff
@@ -50,6 +55,31 @@ void copyElement(sdf::ElementPtr _sdf, const tinyxml2::XMLElement *_xml)
   }
 }
 
+//////////////////////////////////////////////////
+std::string findResourceSdf(const std::string &_path)
```

Comment:
As I mentioned on the `ign-gazebo` PR, I think this could be supported from within the gazebo server, instead of having each client reimplement the logic to find Fuel worlds.